Source/Plugins/Core/com.equella.core/src/com/tle/core/viewcount/service/ViewCountService.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * The Apereo Foundation licenses this file to you under the Apache License,
+ * Version 2.0, (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.tle.core.viewcount.service;
+
+import com.tle.beans.Institution;
+import com.tle.beans.entity.itemdef.ItemDefinition;
+import com.tle.beans.item.ItemKey;
+import com.tle.beans.viewcount.ViewcountAttachment;
+import com.tle.beans.viewcount.ViewcountItem;
+import java.util.List;
+
+public interface ViewCountService {
+  /**
+   * Get the view count of an Item.
+   *
+   * @param itemKey The Item's unique key.
+   */
+  int getItemViewCount(ItemKey itemKey);
+  /**
```

Comment:
I'll only make this whitespace suggestion here, but seeing they're methods can you please leave a line between the method definition and the Javadoc?

```suggestion

  /**
```